eng/common/pipelines/templates/steps/update-docsms-metadata.yml

```diff
@@ -32,6 +32,9 @@ parameters:
   - name: DocValidationImageId
     type: string
     default: ''
+  - name: AdditionalSteps
```

Comment:
Consider using a name like `PreCommitSteps` in case we have to insert more extension points later (not something I think we'll ever do but one doesn't know).